easy/3.js

```diff
@@ -1,11 +1,30 @@
 function minMax(arr) {
-	
+    let howMany = arr.length;
+    let winner = arr[0]; 
+    let loser = arr[howMany-1];
+    
+
+    for(i = 0; i<=howMany; i++){
+        if(winner<arr[i]){
+            winner = arr[i];
+        }
+        if(loser>arr[i]){
+            loser = arr[i];
+        }
+    }
+    
+
+    let returnValueArr = [loser, winner];
+    //console.log(returnValueArr);
+    
+    return returnValueArr;
 }
 
-testSimilar(minMax([14, 35, 6, 1, 34, 54]), [1, 54])

```

Comment:
There are no errors, because you commented my test :(

Reply:
i know. still don't do this exercise.